lib/OpenQA/Shared/Controller/Auth.pm

```diff
@@ -159,7 +159,7 @@ sub _key_auth ($self, $reason, $key) {
           $username, $self->tx->remote_address, $api_key->key, $api_key->secret;
         if (!_is_timestamp_valid($build_tx_timestamp, $timestamp)) {
             $log->debug("$reject_msg, $reason");
-            $reason = 'timestamp mismatch';
+            $reason = 'timestamp mismatch - check localtime vs webui';
```

Comment:
I would avoid the term "localtime" because one might think it refers to the time in your local timezone (as opposed to the timezone used on the web UI host). However, the timezone does not need to match between the hosts. The clocks just need to be generally in sync.

```suggestion
            $reason = 'timestamp mismatch - check whether clocks on the local host and the web UI host are in sync';
```